src/miniscript/mod.rs

```diff
@@ -26,114 +26,124 @@
 
 #[cfg(feature = "serde")]
 use serde::{de, ser};
+use std::marker::PhantomData;
 use std::{fmt, str};
 
 use bitcoin;
 use bitcoin::blockdata::script;
 
 pub mod astelem;
+pub mod context;
```

Comment:
I think we should keep this module private (we publicly rexport some things below)